src/spaces/homspace.jl

```diff
@@ -158,6 +158,47 @@ function compose(W::HomSpace{S}, V::HomSpace{S}) where {S}
     return HomSpace(codomain(W), domain(V))
 end
 
+"""
+    insertunit(W::HomSpace, i::Int=ndims(W) + 1; conj=false, dual=false, preferdomain=false)
```

Comment:
I don't think we use `ndims` in the TensorKit ecosystem, but rather `numind` :-)